README.md

```diff
@@ -11,27 +11,17 @@ One of the more useful cases is flashing other (commercial) energy-monitoring sm
 The focus on the wiki is for plugs that are re-flashable "over the air" for simplicity, but if you're comfortable with soldering (and opening the plug) there are  an incredible number of compatible plugs/devices compatible with ESPHome.
 
 # Usage
-Place the `espsense.h` [custom component](https://esphome.io/custom/custom_component.html) in your ESPHome build directory, and then modify/create your ESPHome YAML definition to include:
-1. the `custom_component` details and lambda, to tell ESPSense which ESPHome sensor to utilize for power data (note: this can also be a [template sensor](https://esphome.io/components/sensor/template.html) that returns a wattage value!)
-2. an `includes:` directive, calling out the `espsense.h` file
-3. a `libraries:` directive, that specifies to include the libraries `ArduinoJson-esphome@5.13.3` and `ESPAsyncUDP` for ESP8266-based devices. **Note**: for ESP32-based devices do *not* include `ESPAsyncUDP`, the core library `AsyncUDP` is utilized instead.
+Modify/create your ESPHome YAML definition to include:
+1. an `external_component` directive, that specifies this component
+2. the `espsense` details, to tell ESPSense which ESPHome sensor(s) to utilize for power data (note: this can also be a [template sensor](https://esphome.io/components/sensor/template.html) that returns a wattage value!)
```

Comment:
```suggestion
2. a top level `espsense` directive, to configure the ESPSense component by specifying which ESPHome sensor(s) to utilize for power data for each plug (note: these can also be [template sensors](https://esphome.io/components/sensor/template.html) that return a wattage value!)
```

Reply:
Documentation has always been my weak point, I agree this is clearer.